components/core/config/config.ts

```diff
@@ -18,7 +18,6 @@ import {
   NzSizeMDSType,
   NzTSType
 } from 'ng-zorro-antd/core/types';
-import { NzTableSortOrder } from 'ng-zorro-antd/table';
```

Comment:
import type { NzTableSortOrder } from 'ng-zorro-antd/table';

Reply:
Still have the same error if using this import

**Entry point ng-zorro-antd/table has a circular dependency on ng-zorro-antd/core/config.**